frontend/src/components/ui/badge.ts

```diff
@@ -79,13 +90,15 @@ export class Badge extends TailwindElement {
                 primary: tw`bg-primary text-neutral-0`,
                 cyan: tw`bg-cyan-50 text-cyan-600`,
                 blue: tw`bg-blue-50 text-blue-600`,
+                violet: tw`bg-violet-50 text-violet-600`,
+                orange: tw`bg-orange-50 text-orange-600`,
                 text: tw`text-blue-500`,
                 "text-neutral": tw`text-neutral-500`,
               }[this.variant],
           this.pill
             ? [
-                tw`min-w-[1.125rem] rounded-full`,
-                this.size === "large" ? tw`px-1.5 py-0.5` : tw`px-1`,
+                tw`min-w-[1.125rem] rounded-full px-1.5`,
+                this.size === "large" && tw`py-0.5`,
```

Comment:
Should we maybe add a little more horizontal padding to pills? The more rounded corners make it feel a lot tighter than the non-rounded versions.